execution/gethexec/express_lane_service_test.go

```diff
@@ -390,26 +400,47 @@ func Test_expressLaneService_sequenceExpressLaneSubmission_outOfOrder(t *testing
 			Transaction:    emptyTx,
 		},
 	}
+	var wg sync.WaitGroup
+	wg.Add(7)
```

Comment:
This is a clever use of wait groups; you're using `wg.Done()` before all calls and after successful calls. My understanding is that there are 5 calls due to there being 5 `messages` and seq num 1 and 2 succeed, giving 7. Is this right? Can you add a short comment about how you derived 7 here and how you're using the wait group?

Reply:
Thanks! I'll add a comment with explanation here